static_src/components/users.jsx

```diff
@@ -176,19 +175,45 @@ export default class Users extends React.Component {
   }
 
   get userInvite() {
-    if (!this.currentUserIsOrgManager) {
+    // When on the org page, only the org manager should see the user invite
+    // form. If not, display notification.
+    if (this.isOrganization && !this.currentUserIsOrgManager) {
       return (
         <PanelDocumentation>
-          Currently, only an org manager can invite users to this { this.entityType } via
-          the dashboard. If the user you want to add is already a member
-          of this organization, you can invite them using the
-          <a href={ cfCliLink }> cloud foundry command line interface. </a>
-          Speak to your org manager if you need to add a user to this { this.entityType } who
-          is not a member of this organization.
+          Only an org manager can new invite users to this organization via
+          the dashboard. Speak to your org manager if you need to add a user
+          to this organization
+        </PanelDocumentation>
+      );
+    }
+    // When on the space page, likewise, an org manager should always see the
+    // invite form. Else, let's dig into what to display.
+    if (this.isSpace && !this.currentUserIsOrgManager) {
+      // if the user is a space manager, let them know that they can invite
+      // existing org users but not new ones.
+      if (this.currentUserIsSpaceManager) {
+        return (
+          <PanelDocumentation>
```

Comment:
In the future, I can imagine there only needing to be one <PanelDocumentation> return statement. All the other content can be compiled as a raw variable.